### PR TITLE
Cake 514 - CMS tracking changes

### PR DESCRIPTION
**Pull request by nmonterroso**
@Wikia/cake 
https://wikia-inc.atlassian.net/browse/CAKE-514

When navigating directly to an action page with the embedded CMS some of the initial JS variables don't get correctly put onto the page. See Wikia/sd-poc#316 for more info

### CODE REVIEW

File: extensions/wikia/ContributionPrototype/ContributionPrototype.setup.php [hunk 1]
```diff
@@ -36,3 +36,28 @@
...
 };
+
+$wgHooks['MakeGlobalVariablesScript'][] = function(&$vars, OutputPage $outputPage) {
+	$title = $outputPage->getTitle();
+	if ($title->getNamespace() === NS_MAIN) {
+		if (preg_match('/(.*)\/(edit|view|history)$/', $title->getText(), $matches)) {
+			$title = Title::newFromText($matches[1]);
+
+			$vars = array_merge($vars, [
+					'wgPageName' => $title->getPrefixedDBkey(),
```
Comment (drsnyder) [inline extensions/wikia/ContributionPrototype/ContributionPrototype.setup.php:47]: Is this right `$title->getPrefixedDBkey()`?

**Comment by drsnyder** [comment]
Minor comment. LGTM

File: extensions/wikia/ContributionPrototype/ContributionPrototype.setup.php (hunk 1 above)
Comment (nmonterroso) [reply extensions/wikia/ContributionPrototype/ContributionPrototype.setup.php:47]: [Yes](https://github.com/Wikia/app/blob/dev/includes/OutputPage.php#L3080)

Comment (drsnyder) [reply extensions/wikia/ContributionPrototype/ContributionPrototype.setup.php:47]: It's mediawiki.

**Pull request merged**